### ISSUE TITLE
[Bug]: FocusZone makes scroll to be trapped even when clicking on the scroll arrow

### ISSUE THREAD

**Issue by dacs30** [microsoft/fluentui#28298]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (24) x64 AMD Ryzen Threadripper PRO 5945WX 12-Cores
    Memory: 91.86 GB / 127.86 GB
  Browsers:
    Edge: Spartan (44.22621.1848.0), Chromium (114.0.1823.51)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/dacs30/pen/JjeRVyp

## To repro
Open the callout from example and try to scroll down using the arrows.
## Actual Behavior
When you have a callout with multiple focuszones (eg a list of message items), the user cannot use the scroll arrows to scroll down because it gets trapped. Making the scroll wheel and keyboard navigation the only option.

## Expected Behavior
User should be able to control the scroll using the button too and not be trapped by focus in only one option

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
